docs/asciidoc/modules/ROOT/pages/export/cypher.adoc

```diff
@@ -65,6 +65,7 @@ The procedures support the following config parameters:
 | awaitForIndexes | Long | 300 | Timeout to use for `db.awaitIndexes` when using `format: "cypher-shell"`
 | saveIndexNames | boolean | false | Save name indexes on export
 | saveConstraintNames | boolean | false | Save name constraints on export
+| multipleRelationshipsWithType | boolean | false | In case of multiple relationships of the same type between two nodes, add a `UNIQUE IMPORT ID REL` property in order to distinguish them when using `MERGE`.
```

Comment:
I can see how for existing users we wouldn't want to break backwards compatibility and that for them it might be good for this to be off by default. And I can also see that for new users it might be better for this to be enabled by default since an export should include multiple relationships.

Could I ask what your thoughts are about setting this to false by default?

Reply:
I just wanted to make sure I understood it right. I think that both approaches have their pros and cons - maybe it's best to leave it like this in dev also for consistency, but if you want to change it in dev then that's OK too.

Reply:
Ok, I'll leave it consistent in dev

Reply:
So the reason for only merging relationships on type is the massive performance impact, it is easily 10x slower to merge on rel-properties. So that's why the default was to focus on fast import as having multiple rels between the same nodes is rarer and the user should be able to configure what's the single id-property on the relationship to minimize the MERGE comparison effort. 
It might benefit from rel-indexes if there are any defined but that's also up to the user.

So from my perspective I would leave the defaults as they were and users that really need this feature should enable it and deal with the the performance impact.

Reply:
Yes, it makes totally sense. I left the default as before